.github/workflows/create-release.yml

```diff
@@ -115,27 +117,48 @@ jobs:
 
       - name: Create git tag
         run: |
-          git tag "${RELEASE_TAG}"
-          git push origin "${RELEASE_TAG}"
+          ./scripts/shell/create_git_tag.sh "${RELEASE_TAG}"
+          
 
-  build-image:
-    name: Build container image
+  wait-for-build:
+    name: Wait for image build job
     needs: [ validate-input-params, create-git-tag ]
     runs-on: ubuntu-latest
     steps:
-      - name: Build image
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with: # release scripts are used from main branch.
+          ref: main
+
+      - name: Setup python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+          cache: 'pip'
+
+      - name: Install requirements
+        run: |
+          pip install -r ./scripts/python/wait-for-commit-check/requirements.txt
+
+      - name: wait for build
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+          GIT_REF: ${{ inputs.name }}
+          GIT_CHECK_RUN_NAME: "build-${{ inputs.name }} / Build image"
```

Comment:
Can you please put a comment here why we decided to use this naming pattern?

Reply:
added